<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Apropos</class>
 <widget class="QMainWindow" name="Apropos">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>271</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>271</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>271</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/resources/icone.ico</normaloff>:/resources/icone.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QGraphicsView" name="imagePreviSat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="cacheMode">
     <set>QGraphicsView::CacheBackground</set>
    </property>
   </widget>
   <widget class="QLabel" name="nomLogiciel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true">PreviSat 3.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="numeroVersion">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Version 5.0.0.11 (20 mai 2012)</string>
    </property>
   </widget>
   <widget class="QLabel" name="informationsCopyright">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Copyright © Astropedia 2005-2023</string>
    </property>
   </widget>
   <widget class="QLabel" name="siteInternet">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">&lt;a href=&quot;http://previsat.sourceforge.net/&quot;&gt;http://previsat.sourceforge.net/&lt;/a&gt;</string>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ok">
    <property name="geometry">
     <rect>
      <x>568</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="description">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>351</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>PreviSat est un logiciel de suivi des satellites artificiels afin de les observer.

 Il permet d'afficher la position des satellites en temps réel ou en mode manuel. PreviSat est capable d'effectuer les prévisions de passage, de calculer les flashs Iridium, ainsi que plusieurs autres calculs de prévisions.

PreviSat est entièrement gratuit!</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dedicace">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Un immense merci à Michel Casabonne pour ses nombreuses suggestions pour PreviSat.</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
